src/main/java/org/reaktivity/nukleus/http_cache/internal/proxy/cache/Cache.java

```diff
@@ -136,34 +136,61 @@ public void put(
         CacheEntry oldCacheEntry = cachedEntries.get(requestUrlHash);
         if (oldCacheEntry == null)
         {
-            CacheEntry cacheEntry = new CacheEntry(
+            final CacheEntry cachedEntry;
+
+            if (request.protocolStackHeader())
```

Comment:
Please rename `request.protocolStackHeader()` to `request.protocolStackEmulated()`.